cost-analyzer/values.yaml

```
@@ -6,6 +6,17 @@ global:
   ##
   zone: ""  # cluster.local
 
+  clusterId: cluster-one  # globally unique cluster name
+
+  ## The name of the Secret containing a bucket config for federated storage.
+  ## The contents should be stored under a key named storage-config.yaml by default.
+  ## Ref: https://www.ibm.com/docs/en/kubecost/self-hosted/2.x?topic=cluster-long-term-federated-storage-configuration
+#  federatedStorage:
+#    existingSecret: federated-storage-config
+#    fileName: federated-store.yaml # set this value to change what the file name for the yaml config is
```

This is a minor suggestion, and can be addressed in a future PR. My preference is for all configs to be explicitly defined in the `values.yaml` file. No configs that are commented out.

Should we instead define `existingSecretName` here? Its default value is an empty string, which in Helm is considered an empty value. Only when a user defines a string here will we assume that a secret exists.

```suggestion
  federatedStorage:
    existingSecretName: ""
    fileName: federated-store.yaml # set this value to change what the file name for the yaml config is
```